tests/virtctl/usbredir.go

```diff
@@ -0,0 +1,271 @@
+//go:build !s390x
```

Comment:
Shouldn't this be handeled by labels instead?

Reply:
Introduced in 5de6a6c2a2aaa8dff939a1bbcb430c1186e1c7c8
Also: 49509be6cb

I'd rather keep it and change in a follow-up PR if you want this changed